.github/ISSUE_TEMPLATE/-bug-report-.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "[Bug Report]"
+about: For noting problems or unexpected behavior
+
+---
+
+**Bug Description**
+Please enter a clear and concise description of what the bug is.
+
+**Reproduction Steps**
+Enter steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+Give a clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Client logs:**
+For issues that make it to the point of reaching the frontend in a browser, please include the javascript logs from that browser. 
+
+**Server logs:**
+For any issue that involved launching the visdom server, please include the server logs:
```

Comment:
You need to see server logs not just for server-launch problems, but also for certain runtime problems. 

It may also be good to remark here that the server doesn't currently log into a particular file (maybe it should?) but simply to stdout.